doc/source/conf.py

```diff
@@ -65,6 +110,10 @@
         '../examples/segmentation',
         '../examples/xx_applications',
     ]),
+    'reference_url'     : {
+            'skimage': None,
+    },
+    'image_scrapers' : ('matplotlib', ImageFileScraper())
```

Comment:
(specifically, here, you shouldn't instantiate the class, just pass the class itself)